litedram/phy/ddr5/s7phy.py

```diff
@@ -44,11 +44,11 @@ def __init__(self, pads, *, iodelay_clk_freq, with_odelay, **kwargs):
         # Note: this should be named sys16x, but using sys8x due to a name hard-coded in BIOS
         assert iodelay_clk_freq in [200e6, 300e6, 400e6]
         iodelay_tap_average = 1 / (2*32 * iodelay_clk_freq)
-        half_sys8x_taps = math.floor(self.tck / (4 * iodelay_tap_average))
-        assert half_sys8x_taps < 32, "Exceeded ODELAYE2 max value: {} >= 32".format(half_sys8x_taps)
+        half_sys4x_taps = math.floor(self.tck / (4 * iodelay_tap_average))
+        assert half_sys4x_taps < 32, "Exceeded ODELAYE2 max value: {} >= 32".format(half_sys4x_taps)
 
         # Registers --------------------------------------------------------------------------------
-        self._half_sys8x_taps = CSRStorage(5, reset=half_sys8x_taps)
+        self._half_sys8x_taps = CSRStorage(5, reset=half_sys4x_taps)
```

Comment:
Mismatched variable name, shouldn't it be `self._half_sys4x_taps`?

Reply:
This value was later removed in  PR #50, so I'll remove it now and rebase #50